inspire/modules/workflows/static/js/workflows/actions/core_approval.js

```diff
@@ -49,6 +50,8 @@ define(
     */
     function CoreApprovalAction() {
 
+      var that = this;
```

Comment:
Ah yeah, cause it's the same logic [here](https://github.com/inspirehep/inspire-next/pull/217/files#diff-7437cf06b1304ee234deb0e71c12d6eaL82). ;)